# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-ac-build gcc-g++-bootstrap-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.64)
./configure --disable-libatomic --disable-libcilkrts --disable-libgomp --disable-libmpx --disable-libquadmath --disable-libsanitizer --disable-libssp --disable-libvtv --disable-lto --disable-multilib --disable-plugin --enable-languages=c++ --with-native-system-header-dir=/include --prefix=

# build
make

# install
install -d $DESTDIR/bin
install host-`./config.guess`/gcc/cpp $DESTDIR/bin
install host-`./config.guess`/gcc/xg++ $DESTDIR/bin/g++
ln -sf g++ $DESTDIR/bin/c++
install host-`./config.guess`/gcc/xgcc $DESTDIR/bin/gcc
ln -sf gcc $DESTDIR/bin/cc
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cassert $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cctype $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cerrno $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cfenv $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cfloat $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/ciso646 $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cinttypes $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/climits $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/clocale $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cmath $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/csignal $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cstdarg $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cstddef $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cstdint $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cstdio $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cstdlib $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cstring $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cwchar $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/ctime $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cwctype $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/algorithm $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/array $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/atomic $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/bitset $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/chrono $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/condition_variable $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/deque $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/forward_list $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/fstream $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/functional $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/future $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/iomanip $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/ios $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/iosfwd $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/iostream $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/istream $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/iterator $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/limits $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/list $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/locale $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/map $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/memory $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/mutex $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/numeric $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/ostream $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/queue $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/random $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/ratio $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/regex $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/set $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/shared_mutex $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/sstream $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/stack $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/stdexcept $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/streambuf $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/string $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/system_error $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/thread $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/tuple $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/type_traits $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/unordered_map $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/unordered_set $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/utility $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/vector $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`/backward
install -m 644 libstdc++-v3/include/backward/auto_ptr.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/backward
install -m 644 libstdc++-v3/include/backward/binders.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/backward
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/algorithmfwd.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/alloc_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/allocated_ptr.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/allocator.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/atomic_base.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/atomic_futex.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/basic_ios.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/basic_ios.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/basic_string.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/basic_string.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/char_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/codecvt.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/concept_check.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/cpp_type_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/deque.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/forward_list.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/forward_list.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/fstream.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/functexcept.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/functional_hash.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/hashtable.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/hashtable_policy.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/ios_base.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/istream.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/list.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/locale_classes.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/locale_classes.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/locale_conv.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/locale_facets.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/locale_facets.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/locale_facets_nonio.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/locale_facets_nonio.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/localefwd.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/memoryfwd.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/move.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/ostream.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/ostream_insert.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/parse_numbers.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/postypes.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/predefined_ops.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/ptr_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/quoted_string.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/random.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/random.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/range_access.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex_automaton.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex_automaton.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex_compiler.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex_compiler.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex_constants.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex_error.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex_executor.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex_executor.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex_scanner.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/regex_scanner.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/shared_ptr.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/shared_ptr_atomic.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/shared_ptr_base.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/sstream.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/std_mutex.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_algo.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_algobase.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_bvector.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_construct.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_deque.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_function.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_heap.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_iterator.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_iterator_base_funcs.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_iterator_base_types.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_list.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_map.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_multimap.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_multiset.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_numeric.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_pair.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_queue.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_raw_storage_iter.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_relops.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_set.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_stack.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_tempbuf.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_tree.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_uninitialized.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_vector.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stream_iterator.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/streambuf.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/streambuf_iterator.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stringfwd.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/uniform_int_dist.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/unique_ptr.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/unordered_map.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/unordered_set.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/uses_allocator.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/vector.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`/debug
install -m 644 libstdc++-v3/include/debug/assertions.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/debug
install -m 644 libstdc++-v3/include/debug/debug.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/debug
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/aligned_buffer.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/alloc_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/atomicity.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/concurrence.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/new_allocator.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/numeric_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/string_conversions.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/type_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/libsupc++/atomic_lockfree_defines.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/libsupc++/cxxabi.h $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/libsupc++/cxxabi_forced.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/libsupc++/exception $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/libsupc++/exception_defines.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/libsupc++/exception_ptr.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/libsupc++/hash_bytes.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/libsupc++/initializer_list $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/libsupc++/nested_exception.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/libsupc++/new $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/libsupc++/typeinfo $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/atomic_word.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/basic_file.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/c++allocator.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/c++config.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/c++io.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/c++locale.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/cpu_defines.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/ctype_base.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/ctype_inline.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/cxxabi_tweaks.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/error_constants.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/gthr-default.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/gthr.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/messages_members.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/opt_random.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/os_defines.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/time_members.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -d $DESTDIR/lib
ln -sf ../bin/cpp $DESTDIR/lib/cpp
install host-`./config.guess`/gcc/libgcc_s.so.1 $DESTDIR/lib
ln -sf libgcc_s.so.1 $DESTDIR/lib/libgcc_s.so
install `./config.guess`/libstdc++-v3/src/.libs/libstdc++.so.6.* $DESTDIR/lib/libstdc++.so.6
ln -sf libstdc++.so.6 $DESTDIR/lib/libstdc++.so
install -d $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtbegin.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtbeginS.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtend.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtendS.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtfastmath.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/libgcc.a $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -d $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/cpuid.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/emmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/float.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/ia32intrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/immintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/mm_malloc.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/mmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/smmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stdarg.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stdbool.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stddef.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stdnoreturn.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/tmmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/unwind.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/x86intrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/xmmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -d $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include-fixed
install -m 644 host-`./config.guess`/gcc/include-fixed/limits.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include-fixed
install -d $DESTDIR/libexec/gcc/`./config.guess`/`cat gcc/BASE-VER`
install host-`./config.guess`/gcc/cc1 $DESTDIR/libexec/gcc/`./config.guess`/`cat gcc/BASE-VER`
install host-`./config.guess`/gcc/cc1plus $DESTDIR/libexec/gcc/`./config.guess`/`cat gcc/BASE-VER`

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/gcc-g++-build.pc << EOF
Name: gcc
Version: 6.5.0
Description: GNU Compiler Collection
Requires: $REQUIRES

devel=\\
/bin/c++ \\
/bin/cc \\
/bin/cpp \\
/bin/g++ \\
/bin/gcc \\
/include/c++/`cat gcc/BASE-VER`/algorithm \\
/include/c++/`cat gcc/BASE-VER`/array \\
/include/c++/`cat gcc/BASE-VER`/atomic \\
/include/c++/`cat gcc/BASE-VER`/bitset \\
/include/c++/`cat gcc/BASE-VER`/cassert \\
/include/c++/`cat gcc/BASE-VER`/cctype \\
/include/c++/`cat gcc/BASE-VER`/cerrno \\
/include/c++/`cat gcc/BASE-VER`/cfenv \\
/include/c++/`cat gcc/BASE-VER`/cfloat \\
/include/c++/`cat gcc/BASE-VER`/chrono \\
/include/c++/`cat gcc/BASE-VER`/ciso646 \\
/include/c++/`cat gcc/BASE-VER`/cinttypes \\
/include/c++/`cat gcc/BASE-VER`/climits \\
/include/c++/`cat gcc/BASE-VER`/clocale \\
/include/c++/`cat gcc/BASE-VER`/cmath \\
/include/c++/`cat gcc/BASE-VER`/condition_variable \\
/include/c++/`cat gcc/BASE-VER`/csignal \\
/include/c++/`cat gcc/BASE-VER`/cstdarg \\
/include/c++/`cat gcc/BASE-VER`/cstddef \\
/include/c++/`cat gcc/BASE-VER`/cstdint \\
/include/c++/`cat gcc/BASE-VER`/cstdio \\
/include/c++/`cat gcc/BASE-VER`/cstdlib \\
/include/c++/`cat gcc/BASE-VER`/cstring \\
/include/c++/`cat gcc/BASE-VER`/ctime \\
/include/c++/`cat gcc/BASE-VER`/cwchar \\
/include/c++/`cat gcc/BASE-VER`/cwctype \\
/include/c++/`cat gcc/BASE-VER`/cxxabi.h \\
/include/c++/`cat gcc/BASE-VER`/deque \\
/include/c++/`cat gcc/BASE-VER`/exception \\
/include/c++/`cat gcc/BASE-VER`/forward_list \\
/include/c++/`cat gcc/BASE-VER`/fstream \\
/include/c++/`cat gcc/BASE-VER`/functional \\
/include/c++/`cat gcc/BASE-VER`/future \\
/include/c++/`cat gcc/BASE-VER`/initializer_list \\
/include/c++/`cat gcc/BASE-VER`/iomanip \\
/include/c++/`cat gcc/BASE-VER`/ios \\
/include/c++/`cat gcc/BASE-VER`/iosfwd \\
/include/c++/`cat gcc/BASE-VER`/iostream \\
/include/c++/`cat gcc/BASE-VER`/istream \\
/include/c++/`cat gcc/BASE-VER`/iterator \\
/include/c++/`cat gcc/BASE-VER`/limits \\
/include/c++/`cat gcc/BASE-VER`/list \\
/include/c++/`cat gcc/BASE-VER`/locale \\
/include/c++/`cat gcc/BASE-VER`/map \\
/include/c++/`cat gcc/BASE-VER`/memory \\
/include/c++/`cat gcc/BASE-VER`/mutex \\
/include/c++/`cat gcc/BASE-VER`/new \\
/include/c++/`cat gcc/BASE-VER`/numeric \\
/include/c++/`cat gcc/BASE-VER`/ostream \\
/include/c++/`cat gcc/BASE-VER`/queue \\
/include/c++/`cat gcc/BASE-VER`/random \\
/include/c++/`cat gcc/BASE-VER`/ratio \\
/include/c++/`cat gcc/BASE-VER`/regex \\
/include/c++/`cat gcc/BASE-VER`/set \\
/include/c++/`cat gcc/BASE-VER`/shared_mutex \\
/include/c++/`cat gcc/BASE-VER`/sstream \\
/include/c++/`cat gcc/BASE-VER`/stack \\
/include/c++/`cat gcc/BASE-VER`/stdexcept \\
/include/c++/`cat gcc/BASE-VER`/streambuf \\
/include/c++/`cat gcc/BASE-VER`/string \\
/include/c++/`cat gcc/BASE-VER`/system_error \\
/include/c++/`cat gcc/BASE-VER`/thread \\
/include/c++/`cat gcc/BASE-VER`/tuple \\
/include/c++/`cat gcc/BASE-VER`/type_traits \\
/include/c++/`cat gcc/BASE-VER`/typeinfo \\
/include/c++/`cat gcc/BASE-VER`/unordered_map \\
/include/c++/`cat gcc/BASE-VER`/unordered_set \\
/include/c++/`cat gcc/BASE-VER`/utility \\
/include/c++/`cat gcc/BASE-VER`/vector \\
/include/c++/`cat gcc/BASE-VER`/backward/auto_ptr.h \\
/include/c++/`cat gcc/BASE-VER`/backward/binders.h \\
/include/c++/`cat gcc/BASE-VER`/bits/algorithmfwd.h \\
/include/c++/`cat gcc/BASE-VER`/bits/alloc_traits.h \\
/include/c++/`cat gcc/BASE-VER`/bits/allocated_ptr.h \\
/include/c++/`cat gcc/BASE-VER`/bits/allocator.h \\
/include/c++/`cat gcc/BASE-VER`/bits/atomic_base.h \\
/include/c++/`cat gcc/BASE-VER`/bits/atomic_futex.h \\
/include/c++/`cat gcc/BASE-VER`/bits/atomic_lockfree_defines.h \\
/include/c++/`cat gcc/BASE-VER`/bits/basic_ios.h \\
/include/c++/`cat gcc/BASE-VER`/bits/basic_ios.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/basic_string.h \\
/include/c++/`cat gcc/BASE-VER`/bits/basic_string.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/char_traits.h \\
/include/c++/`cat gcc/BASE-VER`/bits/codecvt.h \\
/include/c++/`cat gcc/BASE-VER`/bits/concept_check.h \\
/include/c++/`cat gcc/BASE-VER`/bits/cpp_type_traits.h \\
/include/c++/`cat gcc/BASE-VER`/bits/cxxabi_forced.h \\
/include/c++/`cat gcc/BASE-VER`/bits/cxxabi_tweaks.h \\
/include/c++/`cat gcc/BASE-VER`/bits/deque.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/exception_defines.h \\
/include/c++/`cat gcc/BASE-VER`/bits/exception_ptr.h \\
/include/c++/`cat gcc/BASE-VER`/bits/forward_list.h \\
/include/c++/`cat gcc/BASE-VER`/bits/forward_list.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/fstream.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/functexcept.h \\
/include/c++/`cat gcc/BASE-VER`/bits/functional_hash.h \\
/include/c++/`cat gcc/BASE-VER`/bits/hash_bytes.h \\
/include/c++/`cat gcc/BASE-VER`/bits/hashtable.h \\
/include/c++/`cat gcc/BASE-VER`/bits/hashtable_policy.h \\
/include/c++/`cat gcc/BASE-VER`/bits/ios_base.h \\
/include/c++/`cat gcc/BASE-VER`/bits/istream.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/list.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/locale_classes.h \\
/include/c++/`cat gcc/BASE-VER`/bits/locale_classes.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/locale_conv.h \\
/include/c++/`cat gcc/BASE-VER`/bits/locale_facets.h \\
/include/c++/`cat gcc/BASE-VER`/bits/locale_facets.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/locale_facets_nonio.h \\
/include/c++/`cat gcc/BASE-VER`/bits/locale_facets_nonio.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/localefwd.h \\
/include/c++/`cat gcc/BASE-VER`/bits/memoryfwd.h \\
/include/c++/`cat gcc/BASE-VER`/bits/move.h \\
/include/c++/`cat gcc/BASE-VER`/bits/nested_exception.h \\
/include/c++/`cat gcc/BASE-VER`/bits/ostream.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/ostream_insert.h \\
/include/c++/`cat gcc/BASE-VER`/bits/parse_numbers.h \\
/include/c++/`cat gcc/BASE-VER`/bits/postypes.h \\
/include/c++/`cat gcc/BASE-VER`/bits/predefined_ops.h \\
/include/c++/`cat gcc/BASE-VER`/bits/ptr_traits.h \\
/include/c++/`cat gcc/BASE-VER`/bits/quoted_string.h \\
/include/c++/`cat gcc/BASE-VER`/bits/random.h \\
/include/c++/`cat gcc/BASE-VER`/bits/random.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/range_access.h \\
/include/c++/`cat gcc/BASE-VER`/bits/regex.h \\
/include/c++/`cat gcc/BASE-VER`/bits/regex.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/regex_automaton.h \\
/include/c++/`cat gcc/BASE-VER`/bits/regex_automaton.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/regex_compiler.h \\
/include/c++/`cat gcc/BASE-VER`/bits/regex_compiler.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/regex_constants.h \\
/include/c++/`cat gcc/BASE-VER`/bits/regex_error.h \\
/include/c++/`cat gcc/BASE-VER`/bits/regex_executor.h \\
/include/c++/`cat gcc/BASE-VER`/bits/regex_executor.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/regex_scanner.h \\
/include/c++/`cat gcc/BASE-VER`/bits/regex_scanner.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/shared_ptr.h \\
/include/c++/`cat gcc/BASE-VER`/bits/shared_ptr_atomic.h \\
/include/c++/`cat gcc/BASE-VER`/bits/shared_ptr_base.h \\
/include/c++/`cat gcc/BASE-VER`/bits/sstream.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/std_mutex.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_algo.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_algobase.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_bvector.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_construct.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_deque.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_function.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_heap.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_iterator.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_iterator_base_funcs.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_iterator_base_types.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_list.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_map.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_multimap.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_multiset.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_numeric.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_pair.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_queue.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_raw_storage_iter.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_relops.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_set.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_stack.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_tempbuf.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_tree.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_uninitialized.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_vector.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stream_iterator.h \\
/include/c++/`cat gcc/BASE-VER`/bits/streambuf.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/streambuf_iterator.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stringfwd.h \\
/include/c++/`cat gcc/BASE-VER`/bits/uniform_int_dist.h \\
/include/c++/`cat gcc/BASE-VER`/bits/unique_ptr.h \\
/include/c++/`cat gcc/BASE-VER`/bits/unordered_map.h \\
/include/c++/`cat gcc/BASE-VER`/bits/unordered_set.h \\
/include/c++/`cat gcc/BASE-VER`/bits/uses_allocator.h \\
/include/c++/`cat gcc/BASE-VER`/bits/vector.tcc \\
/include/c++/`cat gcc/BASE-VER`/debug/assertions.h \\
/include/c++/`cat gcc/BASE-VER`/debug/debug.h \\
/include/c++/`cat gcc/BASE-VER`/ext/aligned_buffer.h \\
/include/c++/`cat gcc/BASE-VER`/ext/alloc_traits.h \\
/include/c++/`cat gcc/BASE-VER`/ext/atomicity.h \\
/include/c++/`cat gcc/BASE-VER`/ext/concurrence.h \\
/include/c++/`cat gcc/BASE-VER`/ext/new_allocator.h \\
/include/c++/`cat gcc/BASE-VER`/ext/numeric_traits.h \\
/include/c++/`cat gcc/BASE-VER`/ext/string_conversions.h \\
/include/c++/`cat gcc/BASE-VER`/ext/type_traits.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/atomic_word.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/basic_file.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/c++allocator.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/c++config.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/c++io.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/c++locale.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/cpu_defines.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/ctype_base.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/ctype_inline.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/error_constants.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/gthr-default.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/gthr.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/messages_members.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/opt_random.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/os_defines.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/time_members.h \\
/lib/cpp \\
/lib/libgcc_s.so \\
/lib/libstdc++.so \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtbegin.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtbeginS.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtend.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtendS.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtfastmath.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/libgcc.a \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/cpuid.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/emmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/float.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/ia32intrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/immintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/mm_malloc.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/mmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/smmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stdarg.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stdbool.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stddef.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stdnoreturn.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/tmmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/unwind.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/x86intrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/xmmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include-fixed/limits.h \\
/libexec/gcc/`./config.guess`/`cat gcc/BASE-VER`/cc1 \\
/libexec/gcc/`./config.guess`/`cat gcc/BASE-VER`/cc1plus

exec=\\
/lib/libgcc_s.so.1 \\
/lib/libstdc++.so.6
EOF

exit
--- gcc-6.5.0.orig/gcc/config/i386/t-linux64
+++ gcc-6.5.0/gcc/config/i386/t-linux64
@@ -33,6 +33,6 @@
 comma=,
 MULTILIB_OPTIONS    = $(subst $(comma),/,$(TM_MULTILIB_CONFIG))
 MULTILIB_DIRNAMES   = $(patsubst m%, %, $(subst /, ,$(MULTILIB_OPTIONS)))
-MULTILIB_OSDIRNAMES = m64=../lib64$(call if_multiarch,:x86_64-linux-gnu)
+MULTILIB_OSDIRNAMES = m64=../lib$(call if_multiarch,:x86_64-linux-gnu)
 MULTILIB_OSDIRNAMES+= m32=$(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib)$(call if_multiarch,:i386-linux-gnu)
 MULTILIB_OSDIRNAMES+= mx32=../libx32$(call if_multiarch,:x86_64-linux-gnux32)
